batch/batch/worker/worker.py

```diff
@@ -1171,6 +1189,7 @@ async def container_config(self):
                 'uidMappings': [],
                 'gidMappings': [],
                 'resources': {
+                    "devices": [{"allow": False, "access": "rwm"}],
```

Comment:
A short explanation for this:

 - The Nvidia container toolkit hook fails when devices is empty (or missing).  I'm not quite sure why.  I couldn't get crun or the hook to generate an error message (it just has exit code 1).
 - Devices seem to be managed by a allow/deny list.  This now denies all devices by default.  It isn't completely clear what the default behavior was.  The runtime adds some default, required devices (/dev/null, /dev/tty, etc.)  I believe the container runtime hook (based on NVIDIA_VISIBLE_DEVICES set below) finds and attaches the Nvidia GPUs.